selfdrive/car/body/values.py

```diff
@@ -37,5 +37,5 @@ class CAR(Platforms):
   ],
 )
 
-CAR_INFO = {p.config.platform_str: p.config.car_info for p in CAR}
-DBC = {p.config.platform_str: p.config.dbc_dict for p in CAR}
+CAR_INFO = CAR.create_carinfo_map()
+DBC = CAR.create_dbc_map()
```

Comment:
can we entirely skip these lines?

Reply:
DBC is imported directly from values.py in a few places, was trying to make this a minimal PR

Reply:
the plan is to remove these once everything is moved over I'm assuming

Reply:
> the plan is to remove these once everything is moved over I'm assuming
yes, everything should directly reference the PlatformConfig when we are done